pom.xml

```diff
@@ -22,25 +22,9 @@
         </license>
     </licenses>
     <scm>
-        <connection>scm:git:git://github.com/smoope/j2objc-maven-plugin.git</connection>
-        <url>https://github.com/smoope/j2objc-maven-plugin</url>
+        <connection>scm:git:git://github.com/kenwdelong/j2objc-maven-plugin.git</connection>
+        <url>https://github.com/kenwdelong/j2objc-maven-plugin</url>
     </scm>
-    <developers>
```

Comment:
I'd like to keep this section as well for the same reason. If you want, we could add [contribustors](https://maven.apache.org/pom.html#Contributors) section instead